src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.java

```diff
@@ -116,13 +135,37 @@ public ChunkOutputWidget(String chunkId, RmdChunkOptions options,
 
       ChunkDataWidget.injectPagedTableResources();
       
-      frame_.getElement().getStyle().setHeight(
-            expansionState_.getValue() == COLLAPSED ? 
-                  ChunkOutputUi.CHUNK_COLLAPSED_HEIGHT :
-                  ChunkOutputUi.MIN_CHUNK_HEIGHT, Unit.PX);
+      if (chunkOutputSize_ == ChunkOutputSize.Default)
+      {
+         frame_.getElement().getStyle().setHeight(
+               expansionState_.getValue() == COLLAPSED ? 
+                     ChunkOutputUi.CHUNK_COLLAPSED_HEIGHT :
+                     ChunkOutputUi.MIN_CHUNK_HEIGHT, Unit.PX);
+      }
+      else if (chunkOutputSize_ == ChunkOutputSize.Full)
+      {
+         frame_.getElement().getStyle().setTop(0, Unit.PX);
```

Comment:
The compiler might do this automatically but you could probably just make one call to `frame_.getElement().getStyle()` then just used the returned `style` in the rest of the calls.